composer.json

```diff
@@ -63,6 +63,12 @@
 			"@phpcs-compatibility",
 			"@phpstan",
 			"@tests"
+		],
+		"pre-autoload-dump": "Aws\\Script\\Composer\\Composer::removeUnusedServices"
+	},
+	"extra": {
+		"aws/aws-sdk-php": [
+			"S3"
```

Comment:
rekl bych, ze tam nemusi to `S3` byt, viz ten priklad a [note](https://github.com/stobrien89/aws-sdk-php/tree/remove-unused-services/src/Script/Composer#:~:text=NOTE%3A%20S3%2C%20Kms%2C%20SSO%20and%20Sts%20are%20used%20by%20core%20SDK%20functionality%20and%20thus%20are%20unsafe%20for%20deletion.%20They%20are%20excluded%20from%20deletion%20in%20this%20script.) na konci

Reply:
A tady v kodu: https://github.com/aws/aws-sdk-php/blob/master/src/Script/Composer/Composer.php#L69

Reply:
Mne to failuje ked tam nic nieje

Reply:
OK, ja to nezkousel, jen me to zaujalo v dokumentaci 🤷